### ISSUE TITLE
exec bad access on 2.2.0

### ISSUE THREAD

**Issue by lucaconlaq** [zwaldowski/BlocksKit#223]
I've recently updated xcode to 5.1 and BlocksKit from 2.0.0 to 2.2.0.
I want to compile only for 32bit architectures. 

here it's my Podfile:

``` ruby
platform :ios, '6.0'
inhibit_all_warnings!
pod 'BlocksKit', '2.2.0'
pod 'SVProgressHUD'
pod 'AFNetworking', '1.3.1'
pod 'libextobjc/EXTScope', '~> 0.3'

post_install do |installer|
    installer.project.targets.each do |target|
        target.build_configurations.each do |configuration|
            target.build_settings(configuration.name)['ARCHS'] = '$(ARCHS_STANDARD_32_BIT)'
        end
    end
end
```

and I've set the architecture of my project to ARCHS_STANDARD_32_BIT.

since this my application start crashing with exec bad access. 
I've double checked and I haven't updated any other library. If I go back using '2.0.0' everything works perfectly again.

basically 70% of the time I run the application the simulator (32 bit) the application crashes after 3-4 seconds.

here is a screenshot of a crash:
https://www.dropbox.com/s/955ji64hdhmdoe7/Screenshot%202014-03-15%2016.45.17.png
here an other: https://www.dropbox.com/s/6c6k4mpec5xrv0c/Screenshot%202014-03-15%2016.46.57.png

I've experienced the same happening in 2 other applications (same Podfile, same situation)

I know that this is pretty generic, but which other information can I provide you?
